src/SME.SGP.Api/Controllers/RelatorioController.cs

```diff
@@ -47,6 +47,16 @@ public async Task<IActionResult> Frequencia([FromBody] FiltroRelatorioFrequencia
             return Ok(await gerarRelatorioFrequenciaUseCase.Executar(filtroRelatorioFaltasFrequenciaDto));
         }
 
+        [HttpPost("faltas-frequencia-mensal")]
+        [ProducesResponseType(typeof(bool), 200)]
+        [ProducesResponseType(typeof(RetornoBaseDto), 500)]
+        [Permissao(Permissao.FF_C, Policy = "Bearer")]
```

Comment:
Utilizar a permissão criada para esse relatório
```suggestion
        [Permissao(Permissao.RFM_C, Policy = "Bearer")]
```